EnglishTranslation-master/QUEST_LV_0300.tsv

```diff
@@ -1206,297 +1206,297 @@ QUEST_LV_0300_20160310_001205	What do we do with the useless scriptures?{nl}We u
+QUEST_LV_0300_20160310_001351	Eventually, we will have to get rid of the monsters so that the villagers can pick up the corals.{nl}You are the only one I can trust, so I ask you to get rid of it.{nl}
+QUEST_LV_0300_20160310_001352	I hope this clears up...{nl}In case something happens, while you are taking care of it, {nl}I will send a person to look at the status of the monsters.{nl}
+QUEST_LV_0300_20160310_001353	It will be great if we could just drive out the monsters, but...{nl}Something feels uncomfortable.
+QUEST_LV_0300_20160310_001354	Oh, you've come back already?{nl}You are as great as I've heard.
+QUEST_LV_0300_20160310_001355	I knew it. I knew that it wouldn't solve that easily.{nl}It's so strange that many monsters are rushing in within a short time.
+QUEST_LV_0300_20160310_001356	This is not just related to the monsters. There must be a different reason.{nl}I will investigate this seperately. Can you ask people here if they saw something strange?
+QUEST_LV_0300_20160310_001357	You would think the Lord Lieutenant would be the first one to help, but no! {nl}He would rather waste his time threatening us... How strange.{nl}
+QUEST_LV_0300_20160310_001358	If it was just a matter of taxing additional offerings, {nl}why doesn't the Lord Lieutenant take some measures.{nl}Anyway, how did the investigation go?{nl}
+QUEST_LV_0300_20160310_001359	So as the number of monsters increased, the strange smell started to spread.{nl}Hmm...{nl}
+QUEST_LV_0300_20160310_001360	I can't think that the monsters are not related to the smell.{nl}Can you check where that smell is coming from?{nl}
+QUEST_LV_0300_20160310_001361	If we are lucky enough, we might be able to find the source of the smell.{nl}If you find it, please bring it to me. I will get rid of it right away.
+QUEST_LV_0300_20160310_001362	I am counting on you.{nl}Please check the spots where the smell is coming from along the coast.
+QUEST_LV_0300_20160310_001363	The smell and the monsters...{nl}
+QUEST_LV_0300_20160310_001364	Just as I thought! The content of the pouch was the powder that allures monsters in.{nl}Somebody must have deliberately used them to draw the monsters in.{nl}
+QUEST_LV_0300_20160310_001365	If the smell disappears, there will be no reason for the monsters to go crazy.{nl}Can you dig out the same pockets from the pit?
+QUEST_LV_0300_20160310_001366	I will find out more about the pocket in the meantime.{nl}I don't know yet whether it suddenly came or if someone buried it.
+QUEST_LV_0300_20160310_001367	Oh my god... This many...{nl}I can see why this many monsters are coming.{nl}
+QUEST_LV_0300_20160310_001368	Someone is definitely disturbing the coral harvesting.{nl}I don't know why, but he is so evil.
+QUEST_LV_0300_20160310_001369	Yes.{nl}I told you that the Lord Lieutenant is little strange.{nl}
+QUEST_LV_0300_20160310_001370	The Lord Lieutenant going out to defeat the monsters would be quite efficient.{nl}Even if the offerings delivery are delayed a bit.{nl}
```

Comment:
"It would be quite efficient of the Lord Lieutenant for him to go out and defeat the monsters.{nl}Even if this delays the offering delivery a bit."